transaction-scenarios/src/scenarios/non_fungible_resource.rs

```diff
@@ -0,0 +1,405 @@
+                            });
+                            builder
+                                .create_non_fungible_resource(
+                                    OwnerRole::None,
+                                    NonFungibleIdType::Bytes,
+                                    false,
+                                    metadata! {},
+                                    btreemap! {
+                                        Mint => (rule!(allow_all), rule!(deny_all)),
+                                        Burn =>  (rule!(allow_all), rule!(deny_all)),
+                                    },
+                                    Some(entries),
+                                )
+                                .try_deposit_batch_or_abort(config.user_account_1.address)
+                        },
+                        vec![],
+                    )
+                },
+                |core, config, state, result| {
+                    state.bytes_non_fungible_resource = Some(result.new_resource_addresses()[0]);
+                    Ok(())
+                },
+            )
+            .successful_transaction_with_result_handler(
+                |core, config, state| {
+                    core.next_transaction_with_faucet_lock_fee(
+                        "non-fungible-resource-create-ruid",
+                        |builder| {
+                            let mut entries = Vec::new();
+                            entries.push(  ComplexFungibleData {
+                                a: 859,
+                                b: vec!["hi".repeat(50)],
+                                c: AnotherObject {
+                                    f1: btreemap!()
+                                }
+                            });
+                            builder
+                                .create_ruid_non_fungible_resource(
+                                    OwnerRole::None,
+                                    false,
+                                    metadata! {},
+                                    btreemap! {
+                                        Mint => (rule!(allow_all), rule!(deny_all)),
+                                        Burn =>  (rule!(allow_all), rule!(deny_all)),
+                                    },
+                                    Some(entries),
+                                )
+                                .try_deposit_batch_or_abort(config.user_account_1.address)
+                        },
+                        vec![],
+                    )
+                },
+                |core, config, state, result| {
+                    state.ruid_non_fungible_resource = Some(result.new_resource_addresses()[0]);
+                    Ok(())
+                },
+            )
+            .successful_transaction_with_result_handler(
+                |core, config, state| {
+                    core.next_transaction_with_faucet_lock_fee(
+                        "non-fungible-resource-mint-32-nfts",
```

Comment:
Out of interest, what's the ballpark limit of mints / transaction at the moment?

Reply:
It depends on the size of the NFT data. I will create a test to provide the exact number.